eclass/toolchain-funcs.eclass

```diff
@@ -1230,4 +1230,25 @@ tc-get-build-ptr-size() {
 		die "Could not determine CBUILD pointer size"
 }
 
+# @FUNCTION: tc-is-lto
+# @RETURN: Shell true if we are using LTO, shell false otherwise
+tc-is-lto() {
+	local f="${T}/test-lto.o"
+
+	case $(tc-get-compiler-type) in
+		clang)
+			$(tc-getCC) ${CFLAGS} -c -o "${f}" -x c - <<<"" || die
```

Comment:
Mostly just wondering, but why not `</dev/null`.

Reply:
For some absurd reason, I've figured out this would be more portable.

Reply:
Fair enough!

Reply:
> For some absurd reason, I've figured out this would be more portable.

It's not equivalent. `</dev/null` is the empty string, but `<<<""` is an empty line (i.e. a single line feed character).